flutter/lib/l10n/app_en.arb

```diff
@@ -53,6 +53,7 @@
   "errorDialogTitle": "Errors",
   "successDialogTitle": "Success",
   "incorrectDatasetsPath": "Selected datasets directory does not contain files for following benchmarks:",
+  "wrongChecksum": "The checksum of the following files are not matched with one defined in benchmark_setting:",
```

Comment:
I think the following would sound better:
>The following files failed checksum validation

Reply:
Done